doc/guide/admin_slurm.rst

```diff
@@ -15,41 +15,69 @@ Check back with us at the end of 2026.
 Flux Maturity
 =============
 
-Flux is in daily production use as the system workload manager on
-`El Capitan <https://hpc.llnl.gov/hardware/compute-platforms/el-capitan>`_
-(currently in slot 1 of the `TOP500 <https://www.top500.org/>`_
-and its unclassified sister systems at LLNL.  The process of standing up
-those systems brought the Flux team together to address problems of
-scale and stability with unprecedented urgency.  As a result, system
-deployments of Flux on virtually any size system are viable at this point.
-That said, the process also brought into focus missing features, now
-prioritized.  As of December 2025:
-
-- Binary packages are only published internally for the TOSS operating
-  system.  Source RPM packages for RHEL 8 and 9 are manually attached to
-  github releases.  Wider operating system support and binary package
-  distribution are being discussed.
-
-- All nodes of Flux instance must run the same flux broker release version.
-  When flux-core 1.0.0 is released, relaxed version interoperability rules
-  will be published.  For now, rolling upgrades are not possible.
-
-- Support for restarting Flux without perturbing running jobs is expected
-  late 2026.
-
-- Reservation support is expected mid 2026.
-
-- Flux's flexible resource representation is not yet mature.  Meanwhile,
-  hardware locality and binding needs of modern applications can be tricky
-  to satisfy in Flux.
-
-- The system Flux does not preserve job :term:`step` data by default.
-
-- Although Flux has an optional local fair-share accounting system for
-  gathering usage data and setting job priorities, there is not yet support
-  for a site-wide, aggregated accounting system.
-
-- No commercial support or training options.
+The Flux project began around 2012.  Flux has been used for a decade or more
+for managing complex ensembles and workflows at LLNL under Slurm and LSF
+in situations where the traditional workload managers were not up to the task.
+Although Flux was designed from the beginning to replace Slurm and was
+deployed as such on several small systems at LLNL, it did not gain momentum
```

Comment:
Here "momentum" means user-ship at the lab, or wider knowledge about it, fixing all the bugs, or some combination of the above? I think momentum is probably a good word choice, and I primarily want to understand it.

Reply:
Not sure if it's the right word, but I meant to refer to the high quality feedback we started getting from sys admins and users once they were forced to engage because their ability to use/maintain this system depended on being able to use flux effectively.  That started a really strong feedback loop.

Before El Cap I think interest was more casual and despite many attemts to engage sys admins in LC (for example) we didn't get very far with that because they were busy with other things and just not that interested.

Reply:
I think momentum is a good word for that.